src/Test/UnitTests/unit_tests.jl

```diff
@@ -56,6 +56,7 @@ function test_model_solution(model, config;
         constraint_primal = nothing,
         constraint_dual   = nothing
     )
+    config.solve || return
```

Comment:
Here is something that should be addressed by the style guide (https://github.com/JuliaOpt/JuMP.jl/pull/1369): I prefer an explicit
```julia
if config.solve
    return
end
```
over the short-circuiting approach.

Reply:
I agree, this should be discussed. I am ok with both forms but I think @mlubin likes the `||` and `&&` short circuits.
Let's leave it as is for now until we decide of a convention